Luchynska Tetiana/my-todo-list/src/app/todo/todo.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import TodoItem from './todoItem'
+
+@Component({
+  selector: 'app-todo',
+  templateUrl: './todo.component.html',
+  styleUrls: ['./todo.component.css']
+})
+
+export class TodoComponent implements OnInit {
+
+  allowAddingItem = true;
+  
+  todo: TodoItem[] = new Array();
+  nextId = 3;
```

Comment:
Why 3?

Reply:
because my initial array had 3 items. Now id depends from length of array